Sources/SwiftProtobuf/type.pb.swift

```diff
@@ -650,6 +649,11 @@ public struct Google_Protobuf_Option: SwiftProtobuf.Message {
   fileprivate var _storage = _StorageClass.defaultInstance
 }
 
+extension Google_Protobuf_Option {
```

Comment:
What is this change for?  This is a generated file, it shouldn't get manual edits.

Reply:
To workaround a new warning, see Twitter conversation: https://twitter.com/nachosoto/status/954517885375795200

I hope it gets removed...

Reply:
Lets open an issue for this new warnings, as this one case it just the tip of the iceberg, if the warning sticks, we are going to have to change the generator because this will no doubt happen on other things also.